client/landing/stepper/hooks/use-start-url.ts

```diff
@@ -0,0 +1,49 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { getLocaleFromPathname, getLocaleFromQueryParam } from 'calypso/boot/locale';
+import { getLoginUrl } from '../utils/path';
```

Comment:
While looking at a bunch of this code over the last few days, I found two really helpful hooks that I think we should use in this code.
```suggestion
import { useFlowLocale } from 'calypso/landing/stepper/hooks/use-flow-locale';
import { useLoginUrl } from '../utils/path';
```

Maybe it's worth tackling that refactor in a follow-up, but I wanted to flag it regardless.

Reply:
Nice, I've applied the change!